src/modules/recipes/api.ts

```diff
@@ -0,0 +1,89 @@
+const API_BASE_URL = "https://www.themealdb.com/api/json/v1/1";
+
+// Recipe için TypeScript türü
+export interface Recipe {
```

Comment:
Move types from this file to the models.ts file. Make sure that you only have on model (type or interface) defined per thing, that you can import it everywhere where you need to use it.